src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let menList = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    menList = menList.filter(men => Math.ceil(men.died / 100) === century);
+  }
```

Comment:
if we have 'century' - no need to make 2 filters (first one and then again). Choose and do only 1 filtering (the same in other functions)